kornia/augmentation/augmentation3d.py

```diff
@@ -186,3 +187,144 @@ def compute_transformation(self, input: torch.Tensor, params: Dict[str, torch.Te
 
     def apply_transform(self, input: torch.Tensor, params: Dict[str, torch.Tensor]) -> torch.Tensor:
         return F.apply_dflip3d(input, params)
+
+
+class RandomAffine3D(AugmentationBase3D):
+    r"""Random 3D affine transformation of the image keeping center invariant.
+
+    Args:
+        degrees (float or tuple or list): Range of degrees to select from.
+            If degrees is a number, then yaw, pitch, roll will be generated from the range of (-degrees, +degrees).
+            If degrees is a tuple of (min, max), then yaw, pitch, roll will be generated from the range of (min, max).
+            If degrees is a list of floats [a, b, c], then yaw, pitch, roll will be generated from (-a, a), (-b, b)
+            and (-c, c).
+            If degrees is a list of tuple ((a, b), (m, n), (x, y)), then yaw, pitch, roll will be generated from
+            (a, b), (m, n) and (x, y).
+            Set to 0 to deactivate rotations.
+        translate (tuple, optional): tuple of maximum absolute fraction for horizontal
```

Comment:
Why translation, scale and share only in 2 directions instead of 3?

Reply:
translation and shear are documentation errors. translation got 3 direction and shear got actually 6 directions (S_xy, S_xz, S_yx, S_yz, S_zx, Szy). I will update them later.

Scale factor is implemented as isotropic as ```Affine2D``` version.